### DISCUSSION TITLE
Where can I make suggestions for the Chromium project?

### DISCUSSION THREAD

**Discussion by ghost** [Eloston/ungoogled-chromium#1545] [Q&A]
I have suggestions for the Chromium project but I cannot find a place to send one. Their GitHub repo is just a mirror.

**Comment by networkException** [comment]
Chromium is a project maintained and managed by Google, I doubt that they allow outside suggestions to their roadmap. The issue tracker however can be found [here](https://www.chromium.org/for-testers/bug-reporting-guidelines)
